services/grpc/router.subscriptions.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/no-this-in-sfc */
+import { status } from '@grpc/grpc-js'
+import { grpcLog } from '@zap/utils/log'
+
+/**
+ * subscribeHtlcEvents - Call lnd grpc subscribeHtlcEvents method and emit events on updates to the stream.
+ *
+ * @param {object} payload Payload
+ * @returns {object} Grpc Call
+ */
+function subscribeHtlcEvents(payload = {}) {
+  if (this.service.subscribeHtlcEvents) {
+    const call = this.service.subscribeHtlcEvents(payload)
+    call.on('data', data => {
+      grpcLog.debug('HTLC EVENT: %o', data)
```

Comment:
Consider using something like `HTLC DATA EVENT` in order to be able to more easily distinguish between various event types. Sorta like what we do in `lightning.subscriptions`